tests/components/scene/xr-mode-ui.test.js

```diff
@@ -1,12 +1,13 @@
 /* global assert, process, setup, suite, test */
-var entityFactory = require('../../helpers').entityFactory;
+import { entityFactory } from '../../helpers.js';
 
-var UI_CLASSES = ['.a-orientation-modal', '.a-enter-vr-button'];
+var UI_CLASSES = ['.a-orientation-modal', '.a-enter-vr'];
```

Comment:
The `a-hidden` class is applied to the wrapper around the button. The test was incorrect before.